lib/get/util/clone.js

```diff
@@ -6,13 +6,9 @@ module.exports = function clone(node) {
         return node;
     }
 
-    var outValue, i, len;
-    var keys = Object.keys(node);
-    outValue = {};
-    for (i = 0, len = keys.length; i < len; i++) {
-        var k = keys[i];
-        var k0 = k.substr(0, 2);
```

Comment:
Both the `Object.keys` (extra array) and the `substr` (extra string) here are unnecessary.